core/include/detray/navigation/intersection/helix_line_intersector.hpp

```diff
@@ -138,39 +135,16 @@ struct helix_intersector_impl<line2D<algebra_t>, algebra_t> {
             const scalar_type dfds =
                 vector::dot(dtds, w) + vector::dot(t, dwds);
 
-            // x_n+1 = x_n - f(s) / f'(s)
-            s_prev = s;
-            s -= f / dfds;
+            return std::make_tuple(f, dfds);
```

Comment:
Shouldn't this be `detray::tuple`?

Reply:
In principle, yes, but last time I tried structured binding it only worked with ```std::tuple```

Reply:
Alright, I think as long as the CI is happy with this we can keep it like this.